.github/workflows/nightly.yml

```diff
@@ -6,7 +6,79 @@ on:
   workflow_dispatch:
 
 jobs:
-  gradle:
+  # In parallel, we run:
+  #   a) all the tests
+  #   b) the mixed-mode tests
+  #   c) the nightly tests
+  # At the moment the nightly tests include the regular tests, but we do both to get valuable coverage in teamscale.
```

Comment:
I'm not sure I understand the point you're making here. If the `nightly_test` task also runs the regular tests, what coverage data would we miss if we didn't have the new `test` task?